docs/tutorials/gettingstartedexample/gse-gen-basic.md

```diff
@@ -13,16 +13,126 @@
+    | PXIe-1487 (4 In 4 Out) | FPGA Bitfiles\\PXIe_1487_4\_In_4\_Out_Gen.lvbitx |
+
+    > The values on the **Resource** tab of **Configuration Settings** are now similar to the figure below.
+
+    ![Configuration Settings Resource Tab](../../images/PXIe-148X-GenGSE-ResourceTab-BasicSimpleGen.png)
+
+3.  Run the VI to perform a generation using the TDMS file created in the previous tutorial.
+
+4.  The generation GSE example will first load the bitfile and then buffer available data from disk to onboard DRAM. After buffering, the **Waiting for Serializer Setup** indicator will enable to indicate the module is ready for generation.
+
+    ![Generation GSE Front Panel Waiting for Serializer Setup](../../images/PXIe-148X-GenGSE-FrontPanel-Waiting.png)
+
+5.  Click the **Serializer Setup Complete** control button to start generation. Generated images display automatically in the First Display Channel tab.
+
+    ![Configuration Settings Resource Tab](../../images/PXIe-148X-GenGSE-FirstDisplayChannel-BasicSimpleGen.png)
+
+6.  The VI will stop automatically when all data from the TDMS file has been generated.
 
 ## Performing a Simple Generation with GPIO Logging
 
+> Note: The Performing a Simple Generation with GPIO Logging tutorial assumes that all input parameters are still configured as specified in the Performing a Simple Generation tutorial.
+
+1.  Select the **Resource** tab and make the following modifications.
+    - Enable **Log GPIO to Disk**.
+
+    > Note: During the first run of the VI in the Performing a Simple Generation tutorial, the **TDMS File Directory** control was automatically populated with a value pointing to a subfolder (\"TDMS Files\"). This subfolder was automatically created within the project folder to load any existing TDMS files.
+
+    > The values on the **Resource** tab of **Configuration Settings** are now similar to the figure below.
+    
+    ![Configuration Settings Resource Tab Log to Disk](../../images/PXIe-148X-GenGSE-ResourceTab-LogGPIOToDisk.png)
+
+2.  Select the **GPIO** tab and make the following modifications.
+    - Add a GPIO line to the **GPIO to Display** array with the GPIO Bank value set to **Ser0 GPIO** and the GPIO Number set to <font face = "courier new">0</font>. Setting these values enables display of GPIO traffic for the GPIO 0 line on the SI0 channel connected to the camera.
+
+    > The values on the **GPIO** tab of **Configuration Settings** are now similar to the figure below.
+
+    ![Configuration Settings GPIO Tab with Des0 GPIO0](../../images/PXIe-148X-Gen-GPIOTab-DisplaySer0GPIO0.png)
+
+3.  Run the VI, wait for the **Waiting for Serializer Setup** indicator to enable, and  click the **Serializer Setup Complete** control button to start the generation. Results are displayed on the various tabs in the **Data Output** indicator.
+
+4.  GPIO timestamp data is plotted in the **GPIO Timestamps** tab.
```

Comment:
Numbered steps should have specific actions so maybe change to something like "Click the **GPIO Timestamps** tab to view digital waveforms of GPIO timestamp data for all GPIO lines included in the **GPIO to Display** array."

The other option is just to make this a blockquote section under step 3.  Either way is fine with me.

I checked the Acq basic tutorial and it looks like this was missed in that tutorial as well so it could be updated in both places.

Reply:
I like idea 1, fixed.